packages/lesswrong/lib/components.ts

```diff
@@ -578,18 +576,26 @@ importComponent("TagHistoryPage", () => require('../components/tagging/history/T
 importComponent("TagActivityFeed", () => require('../components/tagging/TagActivityFeed'));
 importComponent("TagProgressBar", () => require('../components/tagging/TagProgressBar'));
 importComponent("SingleLineTagUpdates", () => require('../components/tagging/SingleLineTagUpdates'));
-importComponent("SubscribeButton", () => require('../components/tagging/SubscribeButton'));
-importComponent("SubforumSubscribeSection", () => require('../components/tagging/SubforumSubscribeSection'));
-importComponent("SubforumMembersDialog", () => require('../components/tagging/SubforumMembersDialog'));
-importComponent("SubforumMember", () => require('../components/tagging/SubforumMember'));
-importComponent("SubforumNotificationSettings", () => require('../components/tagging/SubforumNotificationSettings'));
+
+importComponent("TagPageRouter", () => require('../components/tagging/TagPageRouter'));
+
+
+// Subforums
 importComponent("TagSubforumPage", () => require('../components/tagging/TagSubforumPage'));
 importComponent("TagSubforumPage2", () => require('../components/tagging/TagSubforumPage2'));
-importComponent("TagPageRouter", () => require('../components/tagging/TagPageRouter'));
 importComponent("RightSidebarColumn", () => require('../components/tagging/RightSidebarColumn'));
 importComponent("SidebarSubtagsBox", () => require('../components/tagging/SidebarSubtagsBox'));
 importComponent("SidebarMembersBox", () => require('../components/tagging/SidebarMembersBox'));
 importComponent("TagSubforumPostsSection", () => require('../components/tagging/TagSubforumPostsSection'));
+importComponent("SubscribeButton", () => require('../components/tagging/SubscribeButton'));
+importComponent("SubforumSubscribeSection", () => require('../components/tagging/SubforumSubscribeSection'));
+importComponent("SubforumMembersDialog", () => require('../components/tagging/SubforumMembersDialog'));
+importComponent("SubforumMember", () => require('../components/tagging/SubforumMember'));
+importComponent("SubforumNotificationSettings", () => require('../components/tagging/SubforumNotificationSettings'));
+importComponent("SubforumNotifications", () => require('../components/form-components/SubforumNotifications'));
+importComponent("TagSubforumButton", () => require('../components/tagging/TagSubforumButton'));
+importComponent("SubforumCommentsThread", () => require('../components/tagging/SubforumCommentsThread'));
+importComponent("SubforumIntroBox", () => require('../components/tagging/subforums/SubforumIntroBox'));
```

Comment:
I started organising these a bit, not relevant to this PR. I'm still planning to do a bigger refactor after the forecasting subforum is launched